cmd/initialize_with_genesis.cpp

```diff
@@ -0,0 +1,258 @@
+
+static void check_rlp_err(rlp::DecodingResult err) {
+    if (err != rlp::DecodingResult::kOk) {
+        throw err;
+    }
+}
+
+static intx::uint256 convert_string_to_uint256(std::string const& str) {
+    intx::uint256 res{0};
+    for(size_t i = 0; i < str.size(); i++) {
+        auto current_digit{str.at(i) - '0'};
+        res += current_digit;
+        res *= 10;
+    }
+    return res/10;
+}
+
+static bool is_hex(std::string const& s) {
+  return s.compare(0, 2, "0x") == 0
+      && s.size() > 2
+      && s.find_first_not_of("0123456789abcdefABCDEF", 2) == std::string::npos;
+}
+
+static uint64_t get_uint_from_field(std::string const& field) {
+    int form = is_hex(field)? 16 : 10;
+    uint64_t res = strtoull(field.c_str(), nullptr, form);
+    if (errno == EINVAL)
+    {
+        SILKWORM_LOG(LogLevel::Error) << field << " not a valid number." << std::endl;
+        throw;
+    }
+    else if (errno == ERANGE)
+    {
+        SILKWORM_LOG(LogLevel::Error) << field << " must be uint64" << std::endl;
+        throw;
+    }
```

Comment:
`throw` alone is used to _rethrow_ a catched exception ... nut here we don't have any exception as strotoull is guaranteed to be exception safe.
Plus, as mentioned elsewhere, there is no exception handling in all the program so throwing will cause a dump of the program